docs/en/compute/Compute-Hive.md

```diff
@@ -182,13 +195,80 @@ Instructions and examples till here illustrate how to use Alluxio as one of the
 tables in Hive, together with other filesystems like HDFS. They do not require to change the global
 setting in Hive such as the default filesystem which is covered in the next section.
 
-## Use Alluxio as the Default Filesystem
+## Advanced Setup
+
+### Add Additional Alluxio client-side properties
+
+There are two ways to specify any Alluxio client properties for Hive queries when
+connecting to Alluxio service:
+
+- Specify the Alluxio client properties in `alluxio-site.properties` and
+ensure that this file is on the classpath of Hive service on each node.
+- Add the Alluxio site properties to `conf/hive-site.xml` configuration file on each node.
+
+#### Example: connect to Alluxio with HA
```

Comment:
The `Example` is confusing here and in the later section for me when first viewing the doc.
Could we remove the `Example` or using other names?

Reply:
ok pulled this to a separate section